test/e2e/storage/testsuites/provisioning.go

```diff
@@ -396,16 +469,17 @@ func TestBindingWaitForFirstConsumerMultiPVC(t StorageClassTest, client clientse
 }
 
 // runInPodWithVolume runs a command in a pod with given claim mounted to /mnt directory.
-func runInPodWithVolume(c clientset.Interface, ns, claimName, nodeName, command string, nodeSelector map[string]string, unschedulable bool) {
+func runInPodWithVolume(c clientset.Interface, ns, claimName, suffix, nodeName, command string, nodeSelector map[string]string, unschedulable bool) {
 	pod := &v1.Pod{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Pod",
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "pvc-volume-tester-",
+			Name: "pvc-volume-tester" + suffix,
 		},
 		Spec: v1.PodSpec{
+			NodeName: nodeName,
```

Comment:
Can we specify the node name as a node selector instead?  Setting NodeName directly in the pod spec will completely bypass the scheduler, and we will miss testing our volume-related scheduling logic.

Reply:
Yes, that's possible with the `kubernetes.io/hostname` label. I'll change it.